src/main/java/frc/robot/ControlPanelSetpoint.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot;
+
+public class ControlPanelSetpoint {
+
+    private static ControlPanelSetpoint instance;
+
+    private AcquisitionPositionSetpoint lastInput;
+
+    private boolean isLeft;
+
+    private ControlPanelSetpoint() {
+        this.lastInput = AcquisitionPositionSetpoint.Storage;
+        OperatorPanel.getInstance().updateIsLeft();
```

Comment:
Operator panel should get the instance of this class, not the other way around.
Also, instead of `ControlPanelSetpoint`, you could call this class `OperatorState`